ext/cairo/rb_cairo_font_face.c

```diff
@@ -108,6 +117,11 @@ rb_cairo_font_face_to_ruby_object (cairo_font_face_t *face)
 static VALUE
 cr_font_face_allocate (VALUE klass)
 {
+#if CAIRO_HAS_FT_FONT
+  FT_Error error = FT_Init_FreeType(&library);
```

Comment:
`FT_Init_FreeType()` may be called with the `library` object two or more times. Is it OK?